src/pages/CreateDao.vue

```diff
@@ -222,6 +222,7 @@ export default defineComponent({
 						approvalProgramBytes: await getCompiledDaoApproval(),
 						clearProgramBytes: await getCompiledDaoClear(),
 						...DAO_CONTRACT_STATE_CONFIG,
+						foreignAssets: [token_id as number],
```

Comment:
why do we need `as number` here?

Reply:
when I assign the type, it can be undefined also (when we initialize the state), but here it is definite that it will exist so to get away with the warning used `as number`